package.json

```diff
@@ -49,14 +49,15 @@
         "build": "tsc -b tsconfig.json",
         "watch": "tsc -b tsconfig.json --watch",
         "lint": "eslint src --ext ts",
+        "clean": "shx rm -rf out node_modules",
         "langium:generate": "langium generate",
         "langium:watch": "langium generate --watch"
     },
     "dependencies": {
         "chevrotain": "^9.1.0",
         "colors": "^1.4.0",
         "commander": "^8.0.0",
-        "langium": "1.0.1",
+        "langium": "~1.2.1",
```

Comment:
You also need to update the `langium-cli` dependency. Also, please remove the `chevrotain`, `vscode-uri` and `vscode-languageserver` dependency. These come from langium themselves.

Also update the `vscode-languageclient` to `^8.1.0`.